src/screens/Home/FloatingActions.js

```diff
@@ -72,8 +72,8 @@ function FloatingActions() {
       disabled: !isSendEnabled,
     },
     {
-      title: t('connect'),
-      iconName: 'wallet-connect',
+      title: t('invest'),
+      iconName: 'invest',
```

Comment:
This needs to be reverted - we're keeping this button as "connect" for now

Reply:
Okay, I'll updated it.